OpenFoodFacts.xcodeproj/project.pbxproj

```diff
@@ -1859,7 +1859,7 @@
 					};
 					95C2650D1E96D55B004212EC = {
 						CreatedOnToolsVersion = 8.3;
-						DevelopmentTeam = ZC9CYWD334;
```

Comment:
Can you keep the Dev Team as is ?

Reply:
Please remove all custom development team information in PRs

Reply:
Ya i'll make sure not to change the development teams, i did that so that i could test it on my phone, will make sure to not repeat the mistake